web/src/server/api/routers/media.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from "zod";
+
+import { env } from "@/src/env.mjs";
+import { getMediaStorageServiceClient } from "@/src/features/media/server/getMediaStorageClient";
+import {
+  createTRPCRouter,
+  protectedProjectProcedure,
+} from "@/src/server/api/trpc";
+import { logger } from "@langfuse/shared/src/server";
+import { TRPCError } from "@trpc/server";
+
+export const mediaRouter = createTRPCRouter({
```

Comment:
Isn't this exactly the same behaviour as the get in `web/src/pages/api/public/media/[mediaId].ts`? Can we do with only one of the two?

Reply:
This file sneaked in, was not ready yet to be part of this PR :)